frontend/src/actions/visualization.js

```diff
@@ -0,0 +1,120 @@
+import { V2 } from "../openapi";
+import { handleErrors } from "./common";
+import config from "../app.config";
+import { getHeader } from "../utils/common";
+
+export const GET_VIZ_CONFIG = "GET_VIZ_CONFIG";
```

Comment:
I see a lot of `viz` instead of `vis` in the code base. I am leaning towards `vis` but I would love to hear what others think.

Reply:
No strong opinion either way, maybe leaning slightly towards VIS?